mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -15,22 +25,27 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		ARG 1 settings
 	METHOD method_16841 getOffsetType ()Lnet/minecraft/class_4970$class_2250;
 	METHOD method_17454 createScreenHandlerFactory (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3908;
+		COMMENT @deprecated Consider calling {@link AbstractBlockState#createScreenHandlerFactory} instead. See {@linkplain AbstractBlock the class javadoc}.
```

Comment:
```suggestion
		COMMENT @deprecated Consider calling {@link AbstractBlockState#createScreenHandlerFactory} instead. See <a href="#deprecated-methods">the class javadoc</a>.
```
I recommend such javadoc (which JDK uses), linking to the exact part we need. Same for other methods.